### ISSUE TITLE
[Resource]: MedSci Skills

### ISSUE THREAD

**Issue by Yoojin-nam** [hesreallyhim/awesome-claude-code#1518]
### Display Name

MedSci Skills

### Category

Agent Skills

### Sub-Category

General

### Primary Link

https://github.com/Aperivue/medsci-skills

### Author Name

Aperivue

### Author Link

https://github.com/Aperivue

### License

MIT

### Description

22 Claude Code skills for the full medical research lifecycle — from PubMed literature search with anti-hallucination citation verification to submission-ready IMRAD manuscripts with reporting compliance audits against STARD, PRISMA, STROBE, and 12 other guidelines. Includes 3 end-to-end demos that each produce complete manuscripts, 300-dpi figures, and compliance reports from public datasets. Built by a radiologist and tested on real publications.

### Validate Claims

Run Demo 1 end-to-end in under 5 minutes:

```bash
git clone https://github.com/Aperivue/medsci-skills.git
cp -r medsci-skills/skills/* ~/.claude/skills/
cd medsci-skills/demo/01_wisconsin_bc
python 01_load_data.py
python 02_analyze.py
```

Then ask Claude Code: `/write-paper` — it generates a full IMRAD manuscript. Follow with `/check-reporting STARD` for a 30-item compliance audit with fix recommendations.

For a quick single-skill test: `/check-reporting any_manuscript.md --guideline STROBE` produces an item-by-item compliance report (PRESENT/PARTIAL/MISSING).

### Specific Task(s)

* Run Demo 1 (Wisconsin BC) — produces manuscript + ROC curves + STARD audit + slides from `sklearn.datasets.load_breast_cancer()`
* Install `search-lit` and search PubMed — verify every citation has a real PMID (anti-hallucination)
* Install `check-reporting` and audit any manuscript against STARD, STROBE, or PRISMA guidelines

### Specific Prompt(s)

* `/check-reporting manuscript.md --guideline STARD`
* `/search-lit "diagnostic accuracy of AI for lung nodule detection" --database pubmed --limit 10`
* `/make-figures prisma --identified 500 --screened 350 --eligible 45 --included 23`

### Additional Comments

Resubmission of #1389 (closed for 7-day cooldown, eligible after April 13, 2026). Updates since original submission: 9 skills expanded to 22, added 3 live end-to-end demos with public datasets, v2.1 pipeline with automatic skill chaining (analyze-stats → make-figures → write-paper → check-reporting). Repository renamed from medical-research-skills to medsci-skills (old URL auto-redirects). All reporting guideline checklists retain original CC licenses. No network requests except public APIs (PubMed, Semantic Scholar, CrossRef).

### Recommendation Checklist

- [x] I have checked that this resource hasn't already been submitted
- [x] It has been over one week since the first public commit to the repo I am recommending
- [x] All provided links are working and publicly accessible
- [x] I do NOT have any other open issues in this repository
- [x] I am primarily composed of human-y stuff and not electrical circuits

**Comment by hesreallyhim** [comment]
A cooldown period is currently in effect for your account. Submitting during an active cooldown extends the restriction.

Please wait at least **14 days** before opening any more submissions.

Please review the [CONTRIBUTING guidelines](https://github.com/hesreallyhim/awesome-claude-code/blob/main/docs/CONTRIBUTING.md) and [pinned issues](https://github.com/hesreallyhim/awesome-claude-code/issues) before your next submission.
